test-suite-data.json

```diff
@@ -623,6 +623,18 @@
     "subpath": null,
     "is_invalid": false
   },
+  {
+    "description": "invalid CPAN distribution name as module name",
+    "purl": "pkg:cpan/GDT/URI::PackageURL",
+    "canonical_purl": "pkg:cpan/GDT/URI::PackageURL",
+    "type": "cpan",
+    "namespace": "GDT",
+    "name": "URI::PackageURL",
+    "version": null,
+    "qualifiers": null,
+    "subpath": null,
+    "is_invalid": true
+  },
```

Comment:
Would it be sensible to also add a failing test for the previously accepted pURLs? e.g.
```
  {
    "description": "invalid CPAN with just the module name",
    "purl": "pkg:cpan/URI::PackageURL",
    "canonical_purl": "pkg:cpan/URI::PackageURL",
    "type": "cpan",
    "namespace": null,
    "name": "URI::PackageURL",
    "version": null,
    "qualifiers": null,
    "subpath": null,
    "is_invalid": true
  },
```
 …and maybe the variants that include a version number?

Reply:
right, the test "invalid CPAN distribution without author" cover only the "distribution" naming. added ;)